src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -522,4 +521,47 @@ object DeltaTable {
     }
   }
 
+  /**
+   * :: Evolving ::
+   *
+   * Check if the provided `path` is the root of a Delta table using the given SparkSession.
+   *
+   * @since 0.4.0
+   */
+  @Evolving
+  def isDeltaTable(sparkSession: SparkSession, path: String): Boolean = {
```

Comment:
Just to be future proof here, I'm wondering if this should be `identifier` instead of `path`. For now, it'll be a path, but once we support tables defined in the HiveMetaStore, we should be able to accept things like:
```
DeltaTable.isDeltaTable(spark, "/some/path")
DeltaTable.isDeltaTable(spark, "delta.`/some/path`")
DeltaTable.isDeltaTable(spark, "some_table")
DeltaTable.isDeltaTable(spark, "some_db.some_table")
DeltaTable.isDeltaTable(spark, "some_catalog.ns1.ns2.some_table") // this is Spark 3.0
```

Reply:
My only argument against `identifier` is that it isn't as clear when looking at code documentation or IDE fill-ins that an identifier is a path in some cases and a table in others. 

With that said, I'm going to go ahead and make the change from `path` -> `identifier` because your point is more effective for the long term. We don't want to have to introduce new functions just to rename params.

Reply:
Yeah, in Spark 3.0, identifier is just any way to identify a table. It can be anything from:
1. Here's a path: `file:/some/path`
2. Here's a metastore table: `my_db.my_table`
3. Here's a catalog and some namespaces: `my_catalog.ns1.ns2.my_table`
It is Spark's job and the configuration of the Spark cluster to figure it out. The documentation should reflect that